src/topology/continuous_function/bounded.lean

```diff
@@ -1018,6 +1020,78 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+/-!
+### Star structures
+
+In this section, if `β` is a normed ⋆-group, then so is the space of bounded
+continuous functions from `α` to `β`, by using the star operation pointwise.
+
+If `𝕜` is normed field and a ⋆-ring over which `β` is a normed algebra and a
+star module, then the space of bounded continuous functions from `α` to `β`
+is a star module.
+
+If `β` is a ⋆-ring in addition to being a normed ⋆-group, then `α →ᵇ β`
+inherits a ⋆-ring structure.
+
+In summary, if `β` is a C⋆-algebra over `𝕜`, then so is  `α →ᵇ β`; note that
+completeness is guaranteed when `β` is complete (see
+`bounded_continuous_function.complete`). -/
+
+section normed_group
+
+variables {𝕜 : Type*} [normed_field 𝕜] [star_ring 𝕜]
+variables [topological_space α] [normed_group β] [star_add_monoid β] [normed_star_monoid β]
+variables [normed_space 𝕜 β] [star_module 𝕜 β]
+
+instance : star_add_monoid (α →ᵇ β) :=
+{ star            := λ f, f.comp star star_normed_group_hom.lipschitz,
+  star_involutive := λ f, ext $ λ x, star_star (f x),
+  star_add        := λ f g, ext $ λ x, star_add (f x) (g x) }
+
+lemma coe_star (f : α →ᵇ β) : ⇑(star f) = star f := rfl
```

Comment:
Maybe I'm missing the point here -- is this statement meant to be
```suggestion
lemma coe_star (f : α →ᵇ β) : ⇑(star f) = star ∘ ⇑f := rfl
```

Reply:
Yes, basically. If you look at the goal, it is `⇑(star f) = star ⇑f`. (note that there is `pi.has_star` which makes the right-hand side meaningful).

Reply:
Aha, I see.  Could you please add a docstring with this explanation?

Reply:
Sorry, one more thing, could you make this a simp-lemma?  (If you do, the linter may complain about the next one being a simp-lemma, in which case you should remove the simp attribute on the next one.)

Reply:
done. I did a local (i.e., file-only) `#lint`, but we'll see if it complains during CI.